packages/babel-plugin-orbit-components/index.js

```diff
@@ -71,6 +70,9 @@ const pathOverwrites = {
   useRandomId: "hooks/useRandomId",
   useRandomIdSeed: { path: "hooks/useRandomId", named: true },
   useFocusTrap: "hooks/useFocusTrap",
+  getBreakpointWidth: "utils/mediaQuery",
```

Comment:
No, it is now needed because now we are actually exporting it in our index file. In the past, only the default export was being exported on the index file.

Reply:
I see. But these adjustments to the `index` file are done in the next commit, right?